WordPress/Jetpack/AppConstants.swift

```diff
@@ -14,6 +14,7 @@ import WordPressKit
     static let shareAppName: ShareAppName = .jetpack
     static let mobileAnnounceAppId = "6"
     @objc static let eventNamePrefix = "jpios"
+    @objc static let explatPlatform = "wpios"
```

Comment:
Should this still be `wpios` even though it's in the Jetpack file?

Reply:
That's intentional; it's the workaround to make ExPlat work in Jetpack. See p1682352977248619-slack-C04M1NX8YCR

Reply:
Perfect, thank you for confirming @salimbraksa